### PR TITLE
Window header bar display to right

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

**Comment by smk762** [comment]
Failure to launch in Linux
```
[2021-06-24 13:55:07.701] [info] [main.prerequisites.hpp:381] Installing qt_message_handler
[2021-06-24 13:55:07.703] [info] [main.prerequisites.hpp:386] SSL: true OpenSSL 1.1.1g  21 Apr 2020 OpenSSL 1.1.1f  31 Mar 2020
[13:55:07] [info] [global.utilities.cpp:256] [382148]: Logger successfully initialized
[13:55:07] [info] [main.prerequisites.hpp:154] [382148]: connecting signal SIGABRT to the signal handler
[13:55:07] [info] [main.prerequisites.hpp:256] [382148]: Init timezone db
[13:55:07] [info] [main.prerequisites.hpp:188] [382148]: libsodium successfully initialized
[13:55:07] [info] [main.prerequisites.hpp:194] [382148]: cleaning previous mm2 instance
[13:55:07] [info] [main.prerequisites.hpp:179] [382148]: wally successfully initialized
mm2: no process found
[13:55:07] [info] [main.prerequisites.hpp:279] [382148]: Checking for setup default themes - theme_path: /home/smk762/.atomic_qt/themes original_theme_path: /home/smk762/Downloads/atomicdex-desktop-ubuntu-qt-5-15-2-a9a0177.zip/atomicdex-desktop-linux-a9a0177/AntaraAtomicDexAppDir/usr/share/assets/themes
[13:55:07] [info] [main.prerequisites.hpp:297] [382148]: Checking for setup default logo - logo_path: /home/smk762/.atomic_qt/logo original_logo_path: /home/smk762/Downloads/atomicdex-desktop-ubuntu-qt-5-15-2-a9a0177.zip/atomicdex-desktop-linux-a9a0177/AntaraAtomicDexAppDir/usr/share/assets/logo
[13:55:07] [info] [main.prerequisites.hpp:314] [382148]: Checking for settings ini reconfiguration
[13:55:07] [info] [main.prerequisites.hpp:214] [382148]: initializing high dpi support
[13:55:07] [info] [main.prerequisites.hpp:336] [382148]: reconfiguration for settings finished

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6129:18: Missing name of pseudo-class

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6134:20: Missing name of pseudo-class

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6151:17: Missing name of pseudo-class

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6166:18: '-gtk-icon-effect' is not a valid property name

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6179:16: Missing name of pseudo-class

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6184:16: Missing name of pseudo-class

(atomicdex-desktop:382148): Gtk-WARNING **: 13:55:07.766: Theme parsing error: gtk.css:6200:15: Missing name of pseudo-class
[13:55:07] [debug] [main.prerequisites.hpp:224] [382148]: physical dpi: 81.59734830387005
[13:55:07] [debug] [main.prerequisites.hpp:225] [382148]: logical dpi: 96
[13:55:07] [debug] [main.prerequisites.hpp:230] [382148]: height: 1080
[13:55:07] [debug] [main.prerequisites.hpp:224] [382148]: physical dpi: 69.83486077998207
[13:55:07] [debug] [main.prerequisites.hpp:225] [382148]: logical dpi: 96
[13:55:07] [debug] [main.prerequisites.hpp:227] [382148]: scale: 1
[13:55:07] [debug] [main.prerequisites.hpp:230] [382148]: height: 1080
[13:55:07] [debug] [main.prerequisites.hpp:227] [382148]: scale: 1
[13:55:07] [info] [main.prerequisites.hpp:242] [382148]: dpi settings finished
[13:55:07] [warning] [internet.checker.service.cpp:190] [382148]: mm2 system not available skipping internet connectivity with it
[13:55:07] [info] [ip.checker.service.cpp:96] [382148]: ip_service_checker created
[13:55:07] [info] [raw.mm2.coins.cfg.hpp:237] [382148]: parse_raw_mm2_coins_file
[13:55:07] [info] [raw.mm2.coins.cfg.hpp:262] [382148]: successfully parsed: /home/smk762/.atomic_qt/0.5.0/configs/coins.json
[13:55:07] [info] [raw.mm2.coins.cfg.hpp:263] [382148]: coins size mm2: 300
[13:55:07] [info] [mm2.service.cpp:253] [382148]: mm2_service created
[13:55:07] [info] [qt.settings.page.cpp:62] [382148]: settings_page created
[13:55:07] [info] [qt.portfolio.page.cpp:38] [382148]: portfolio_page created
[13:55:07] [info] [orderbook.scanner.service.cpp:29] [382148]: orderbook_scanner_service created
[13:55:07] [warning] [rpc.best.orders.cpp:44] [382148]: best orders result not available yet - probably seed node unsync
[13:55:07] [info] [band.provider.cpp:54] [382148]: start fetching oracle
[13:55:07] [info] [coingecko.provider.cpp:15] [382148]: coingecko_provider created
[13:55:07] [info] [qt.download.manager.cpp:16] [382148]: qt_download_manager created
[13:55:07] [error] [self.update.service.cpp:268] [382148]: stoi
[13:55:07] [info] [coingecko.wallet.charts.cpp:39] [382148]: coingecko_wallet_charts_service created
[13:55:07] [info] [qt.orders.widget.cpp:33] [382148]: qt_orders_widget created
[13:55:07] [info] [app.cpp:477] [382148]: connecting signals
[13:55:07] [info] [qt.addressbook.page.cpp:52] [382148]: connecting addressbook signals
[13:55:07] [info] [qt.wallet.manager.cpp:143] [382148]: Retrieve wallet name: cswm
[13:55:07] [info] [qt.wallet.manager.cpp:53] [382148]: new wallet name: 
[13:55:07] [info] [app.cpp:357] [382148]: application created
[13:55:07] [info] [main.prerequisites.hpp:354] [382148]: file name settings: /home/smk762/.atomic_qt/0.5.0/configs/cfg.ini
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings CurrentTheme already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings ThemePath already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings AutomaticUpdateOrderBot already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings AvailableLang already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings CurrentLang already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings 2FA already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings MaximumNbCoinsEnabled already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings DefaultTradingMode already exist - skipping
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings FontMode already exist - skipping
[13:55:07] [warning] [app.cpp:300] [382148]: AutomaticUpdateOrderBot is false, ignoring the service
[13:55:07] [info] [qt.portfolio.page.cpp:189] [382148]: new chart category: OneMonth
[13:55:07] [info] [main.prerequisites.hpp:351] [382148]: Settings WalletChartsCategory already exist - skipping
[13:55:08] [info] [qt.settings.page.cpp:114] [382148]: Locale before parsing AtomicDEX settings: en_AU
[13:55:08] [info] [qt.settings.page.cpp:116] [382148]: Locale after parsing AtomicDEX settings: en_US
[13:55:08] [info] [qt.settings.page.cpp:122] [382148]: Post lang changed
[13:55:08] [info] [main.prerequisites.hpp:436] [382148]: post set_qt_app
[13:55:08] [info] [main.prerequisites.hpp:454] [382148]: QML Enum created
[13:55:08] [info] [main.prerequisites.hpp:478] [382148]: QML context properties created
[13:55:08] [info] [main.prerequisites.hpp:485] [382148]: Qaterial type created
[13:55:08] [info] [main.prerequisites.hpp:493] [382148]: QML singleton created
[13:55:08] [info] [main.prerequisites.hpp:506] [382148]: Load qml engine
[13:55:08] [warning] [main.prerequisites.hpp:126] [382148]: QQmlApplicationEngine failed to load component
[13:55:08] [warning] [main.prerequisites.hpp:126] [382148]: qrc:/atomic_defi_design/qml/main.qml:140:17: Property value set multiple times
[13:55:08] [info] [main.prerequisites.hpp:521] [382148]: qml engine successfully loaded
[13:55:08] [info] [app.cpp:162] [382148]: Launch the application
[13:55:08] [debug] [app.cpp:623] [382148]: will quit app, prevent all threading event
[13:55:08] [error] [self.update.service.cpp:268] [382148]: stoi
[13:55:08] [info] [mm2.service.cpp:286] [382148]: destroying mm2 service...
[13:55:08] [info] [mm2.service.cpp:290] [382148]: mm2 signals successfully disconnected
[13:55:08] [info] [mm2.service.cpp:339] [382148]: mm2 service fully destroyed
[13:55:08] [info] [mm2.service.cpp:315] [382148]: mm2 didn't stop yet with rpc stop, stopping process manually
[13:55:08] [info] [main.prerequisites.hpp:250] [382148]: wally successfully cleaned
[13:55:08] [info] [coingecko.provider.cpp:24] [382148]: coingecko_provider destroyed
[13:55:08] [info] [coingecko.wallet.charts.cpp:44] [382148]: coingecko_wallet_charts_service destroyed
[13:55:08] [info] [qt.orders.widget.cpp:35] [382148]: qt_orders widget destroyed
```

File: atomic_defi_design/qml/main.qml [hunk 1]
```diff
@@ -120,33 +120,25 @@ DexWindow {
...
+            DexLabel {
+                text: " | "
+                opacity: .1
+                font.family: 'Montserrat'
+                font.weight: Font.Medium
+                visible: _label.visible
+                color: app.globalTheme.foregroundColor
+                anchors.verticalCenter: parent.verticalCenter
+                leftPadding: 2
+                visible: !isOsx
```
Comment (smk762) [inline atomic_defi_design/qml/main.qml:140]: duplicate property value assignment

Comment (tonymorony) [reply atomic_defi_design/qml/main.qml:140]: @ssakone please check ^^

Comment (ssakone) [reply atomic_defi_design/qml/main.qml]: sorry again :/

**Review by smk762 — APPROVED** [review]
looks good, thanks!

**Pull request merged**